bzt/modules/apiritif/generator.py

```diff
@@ -933,6 +933,8 @@ def _gen_webdriver(self):
 
         browser = self._check_platform()
 
+        body.append(self._get_selenium_options())
+
         if browser == 'firefox':
```

Comment:
refactoring outdated as I've moved logic to '_get_selenium_options()'

Reply:
[https://github.com/Blazemeter/taurus/pull/1475/files#diff-f2b792ae37b5e0585f5a21cfaa1337efb79dd12b25d461302c2e95314e3bb7daR958](url)

Reply:
no, you should generate all options config before create a webdriver instance.
see example for chrome now:
```
 body.extend(self._get_chrome_options() + [self._get_chrome_webdriver()])
 body.append(self._get_selenium_options(browser))
 ```
 
 In python it will look like:
 ```
 options = ChromeOptions()
 driver = ChromeDriver(options)
 options.ignore_local_proxy()
 ```